### PR TITLE
risk factor

### PR DESCRIPTION
**Pull request by niradler**

### CODE REVIEW

File: src/main/kotlin/com/bridgecrew/CheckovResult.kt [hunk 1]
```diff
@@ -1,9 +1,28 @@
 package com.bridgecrew
 
 import com.google.gson.Gson
+import com.google.gson.annotations.SerializedName
+import com.google.gson.reflect.TypeToken
 
 val gson = Gson()
 
+//data class RiskFactors(
```
Comment (saarschweid) [inline src/main/kotlin/com/bridgecrew/CheckovResult.kt:9]: remove this

File: src/main/kotlin/com/bridgecrew/CheckovResult.kt [hunk 2]
```diff
@@ -15,7 +34,7 @@ data class VulnerabilityDetails(
         val lowest_fixed_version: String?,
         val published_date: String?,
         val vector: String?,
-//        val risk_factors: List<String>?, // TODO - fix after Saar's team fixes their side
```
Comment (saarschweid) [inline src/main/kotlin/com/bridgecrew/CheckovResult.kt:18]: remove this

File: src/main/kotlin/com/bridgecrew/services/ResultsCacheService.kt [hunk 3]
```diff
@@ -79,7 +79,7 @@ class ResultsCacheService(val project: Project) {
                                 result.vulnerability_details.vector,
                                 result.vulnerability_details.id,
                                 result.file_path,
-                                null, // TODO - fix after Saar's team fixes their side
```
Comment (saarschweid) [inline src/main/kotlin/com/bridgecrew/services/ResultsCacheService.kt:82]: remove this

File: src/main/kotlin/com/bridgecrew/CheckovResult.kt (hunk 1 above)
Comment (niradler) [reply src/main/kotlin/com/bridgecrew/CheckovResult.kt]: removed

**Review by saarschweid — APPROVED** [review]

**Pull request merged**